index.js

```diff
@@ -59,20 +58,53 @@ async function discoverPeersAndRunServer() {
   const ipTypes = {
     public: "PublicIpAddress",
     private: "PrivateIpAddress"
-  }
+  };
 
   let peers = [];
 
   try {
-    const AWS_REGION = "us-west-2";
+    const AWS_REGION = process.env.AWS_REGION;
     const ec2 = new AWS.EC2({ region: AWS_REGION });
+    const autoscaling = new AWS.AutoScaling({ region: AWS_REGION });
+
+    const {
+      AutoScalingGroups
+    } = await autoscaling.describeAutoScalingGroups().promise();
```

Comment:
scaling group name should be ENV also, we can't assume all scaling groups are gundb

Reply:
added beanstalk environment name as ENV to get peers by it